build.gradle.kts

```diff
@@ -88,15 +88,9 @@ val build by tasks.named("build") {
   dependsOn(securityTest, dataRestTest, txTest, awsTest)
 }
 
-val sourcesJar by tasks.registering(Jar::class) {
-  archiveClassifier.set("sources")
-  from(sourceSets.main.get().allJava)
-}
-
-val javadocJar by tasks.registering(Jar::class) {
-  dependsOn("dokka")
-  archiveClassifier.set("javadoc")
-  from(buildDir.resolve("dokka"))
+java {
+  withJavadocJar()
```

Comment:
We need a way to keep the `dependsOn("dokka")`.

Reply:
Ok

Reply:
I'm afraid it's not enough. The generated javadoc jar is empty.

Reply:
Hello, in fact. For now it seems that the generation of doc with gradle 6 withJavadocJar does not work well with dokka. I think at first we will keep the generation task that you have implemented.

Reply:
Ok. Thanks.